portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+/*
+ * Comment class stores message and sender.
+ */
+
+ public class Comment {
+    private String sender;
+    private String text;
+
+    public Comment(String sender, String text) {
```

Comment:
Generally when you overload a function the additional arguments go last to avoid confusion. So the signature here would be as suggested
```suggestion
    public Comment(String text, String sender) {
```